packages/jest-config/src/normalize.js

```diff
@@ -146,6 +146,11 @@ function normalize(config, argv) {
           config.moduleNameMapper,
         );
       }
+      // Don't show deprecation warnings if the setting comes from the preset.
```

Comment:
the change in this file is entirely unrelated. I apologize. It just makes the deprecation warning go away when using a preset because it isn't actionable anyway unless you are the author of the preset.